### DISCUSSION TITLE
显存占用过大

### DISCUSSION THREAD

**Discussion by BruceWy817** [Tencent/libpag#2505] [Q&A]
[模糊测试文件夹.zip](https://github.com/user-attachments/files/17178708/default.zip)
![9B540C0228148BB53AA46C36450253D0](https://github.com/user-attachments/assets/0895f3e2-4f2e-4263-bd7e-21b5630c1273)

**Comment by CodeJhF** [comment]
抱歉这条当时没有被跟进到，是我们漏了。

如果你还在使用 libpag 且问题仍然存在，麻烦回复一下当前版本号（4.4 / 4.5 系列）和最小复现 demo，我们会优先跟进；如果不再需要跟进，本条就先关闭了。（stale-close）
